src/queries/labelText.ts

```diff
@@ -10,15 +9,28 @@ import type {
   QueryAllByQuery,
   QueryByQuery,
 } from './makeQueries';
+import { TextMatchOptions } from './text';
+
+const getNodeByLabelText = (
+  node: ReactTestInstance,
+  text: TextMatch,
+  options: TextMatchOptions = {}
+) => {
+  const { exact, normalizer } = options;
+  return matches(text, node.props.accessibilityLabel, normalizer, exact);
+};
 
 const queryAllByLabelText = (
   instance: ReactTestInstance
-): ((text: TextMatch) => Array<ReactTestInstance>) =>
-  function queryAllByLabelTextFn(text) {
+): ((
+  text: TextMatch,
+  queryOptions?: TextMatchOptions
+) => Array<ReactTestInstance>) =>
+  function queryAllByLabelTextFn(text, queryOptions?: TextMatchOptions) {
     return instance.findAll(
       (node) =>
         typeof node.type === 'string' &&
-        matchStringProp(node.props.accessibilityLabel, text)
+        getNodeByLabelText(node, text, queryOptions)
```

Comment:
Could you also tests for the label text with `exact: false` please?

Reply:
Have added in latest commits.